Core.UnitTests/Implementation/PipelineSettingsTest.cs

```diff
@@ -26,10 +26,9 @@ public class PipelineSettingsTest
     [Test]
     public void Initialization ()
     {
-      var settings = new PipelineSettings (true, "keyFile", true, "assemblyDirectory", "assemblyName{counter}suffix", 2);
+      var settings = new PipelineSettings (true, "keyFile", "assemblyDirectory", "assemblyName{counter}suffix", 2);
```

Comment:
Please document breaking change ticket for changes to PipelineSettings. It's a public API.
ctor-change, removed members